From aa7896080b2c226f75e2f0c7e48501ffafc6b665 Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Wed, 18 Mar 2020 13:26:37 +0200
Subject: [PATCH] [SW-10047] goya: pldm: pll: remove all config data

The PLLs configuration handling is moving to U-Boot in order to minimize
the preboot image.
The Goya PLDM PLLs configuration seems not needed anymore anyway.
Remove the PLDM PLLs handling from preboot.

Change-Id: I0106ad2ef33eb3c5fa23cd99f7f0c6df99af3ed4
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/goya/pldm.c b/goya/pldm.c
index 5ddd4b1..7391714 100644
--- a/goya/pldm.c
+++ b/goya/pldm.c
@@ -5,8 +5,6 @@
  * SPDX-License-Identifier:     GPL-2.0+
  */
 
-#include <stdbool.h>
-
 #include <preboot.h>
 #include <pll.h>
 #include <uart.h>
@@ -14,28 +12,6 @@
 #include "common.h"
 
 #define SYS_CLOCK_FREQUENCY	150000ULL
-
-static struct hl_pll_cfg goya_pll_cfg[] = {
-	{
-		.name = "PCI", .base = mmPSOC_PCI_PLL_BASE,
-		.nr = 4, .nf = 343, .od = 3, .nb = 171,
-		.div_factor = { 0, 4, 8, 85 },
-		.div_sel = { HL_PLL_PLL_CLK,
-			     HL_PLL_DIV_PLL,
-			     HL_PLL_DIV_PLL,
-			     HL_PLL_DIV_PLL
-		},
-	},
-};
-
-int hl_pll_get_cfg(struct hl_pll_cfg **hl_pll_cfg)
-{
-	int pll_num = ARRAY_SIZE(goya_pll_cfg);
-
-	*hl_pll_cfg = goya_pll_cfg;
-
-	return pll_num;
-}
 
 u32 hl_skip_pcie_init(void)
 {
